server/models/Roommate.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require("mongoose")
+
+const schema = mongoose.Schema({
+    firstName: { type:String, required: true },
+    lastName: { type:String, required: true },
+    phoneNum: { type:Number, required: true },
```

Comment:
Employee's first name, last name, and phone number should already be accessible from personal info collection.

Reply:
Thanks for the comment. It's resolved. Kindly take a look!